test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -47,8 +47,8 @@ if (cluster.isMaster) {
     address: 'localhost',
     port: common.PORT,
     exclusive: false
-  }, function() {
-    socket2.bind({port: common.PORT + 1, exclusive: true}, function() {
+  }, () => {
+    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
```

Comment:
I thought the preferred style in core was to not have spaces inside of the curly braces. I could be wrong though.